src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -712,12 +704,14 @@ namespace ExhaustAirSystemManager {
         // Calculate a zonehvac exhaust control system
 
         auto &thisExhCtrl = state.dataZoneEquip->ZoneExhaustControlSystem(ZoneHVACExhaustControlNum);
+
         int InletNode = thisExhCtrl.InletNodeNum;
         int OutletNode = thisExhCtrl.OutletNodeNum;
         auto &thisExhInlet = state.dataLoopNodes->Node(InletNode);
         auto &thisExhOutlet = state.dataLoopNodes->Node(OutletNode);
         Real64 MassFlow = state.dataLoopNodes->Node(InletNode).MassFlowRate;
```

Comment:
You've already defined `thisExhInlet`, use it.

Reply:
Thanks! Now it is:
```
        Real64 MassFlow = thisExhInlet.MassFlowRate;
```